### PR TITLE
test: handle popovers in e2e test

### PR DESCRIPTION
**Pull request by tormoseng**
The performance test script needs to handle the on-behalf-of popover. Added a step to close a popover if it exists.

NB! Added test-id to the popover component.

### CODE REVIEW

File: src/components/popover/PopOver.tsx [hunk 1]
```diff
@@ -50,9 +50,9 @@ export const PopOver = ({
       }
       backgroundStyle={style.backdrop}
     >
-      <View style={style.heading}>
+      <View style={style.heading} testID="popover">
```
Comment (marius-at-atb) [inline src/components/popover/PopOver.tsx:53]: This testID is not used as far as I can see?

File: e2e/test/utils/app.helper.ts [hunk 2]
```diff
@@ -44,6 +44,17 @@ class AppHelper {
     await driver.saveScreenshot(`${screenshotsFolder}/${fileName}.png`);
   }
 
+  /**
+   * Remove the popover if it exists
+   */
+  async removePopover() {
+    const popoverId = `//*[@resource-id="closePopover"]`;
```
Comment (marius-at-atb) [inline e2e/test/utils/app.helper.ts:51]: Was this meant to be called `closePopoverId`?

**Review by marius-at-atb — APPROVED** [review]
Looks good, see a couple of minor comments

File: src/components/popover/PopOver.tsx (hunk 1 above)
Comment (tormoseng) [reply src/components/popover/PopOver.tsx:53]: That's true, but I added it to later be able to add tests for popovers, as well :-)

File: e2e/test/utils/app.helper.ts (hunk 2 above)
Comment (tormoseng) [reply e2e/test/utils/app.helper.ts]: True! Fixed.

File: src/components/popover/PopOver.tsx (hunk 1 above)
Comment (marius-at-atb) [reply src/components/popover/PopOver.tsx]: Aha 😄

Comment (tormoseng) [reply src/components/popover/PopOver.tsx]: A second thought, if I'll add popover tests I rather need a test-id for the specific popover text. So I'll remove this and rather add proper test-id later 👍

**Pull request merged**